### PR TITLE
Individual pages

### PR DESCRIPTION
**Pull request by SaulCM**
Added individual pages and solved checkstyle errors.

### CODE REVIEW

File: src/main/java/api/ApiManager.java [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package api;
 
+import static io.restassured.RestAssured.given;
+
```
Comment (GustavoZHA) [inline src/main/java/api/ApiManager.java:4]: Delete extra line

File: src/main/java/api/salesforce/Authentication.java [hunk 2]
```diff
@@ -1,11 +1,11 @@
 package api.salesforce;
 
+import static io.restassured.RestAssured.given;
+
```
Comment (GustavoZHA) [inline src/main/java/api/salesforce/Authentication.java:4]: Delete line too

File: src/main/java/core/selenium/WebElementAction.java [hunk 3]
```diff
@@ -90,6 +103,110 @@ public String getTextOfElementByField(final String field) {
...
+     * @param selector represents a selector
+     * @return String represents the text from the web element
+     */
+    public String getText(final By selector) {
+        WebDriverManager.getInstance().getWait().until(ExpectedConditions.visibilityOf(WebDriverManager
+                .getInstance().getWebDriver().findElement(selector)));
+        return WebDriverManager.getInstance().getWebDriver().findElement(selector).getText();
+    }
+
+
```
Comment (GustavoZHA) [inline src/main/java/core/selenium/WebElementAction.java:118]: Delete line

File: src/main/java/api/ApiManager.java (hunk 1 above)
Comment (SaulCM) [reply src/main/java/api/ApiManager.java:4]: It is required by checkstyle regarding static imports.

File: src/main/java/salesforce/ui/pages/lightning/individuals/IndividualRecordPage.java [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+/**
+ * Page Object Model for the salesforce individual record page.
+ */
+public class IndividualRecordPage extends BasePage {
+
+    private By nameHeaderLabel = By.cssSelector("h1 .uiOutputText");
+    private By editHeaderButton = By.cssSelector("div[title=\"Edit\"]");
+    private By deleteHeaderButton = By.cssSelector("div[title=\"Delete\"]");
+    private By cloneHeaderButton = By.cssSelector("div[title=\"Clone\"]");
+
```
Comment (GustavoZHA) [inline src/main/java/salesforce/ui/pages/lightning/individuals/IndividualRecordPage.java:25]: organize and delete blank lines

File: src/main/java/api/salesforce/Authentication.java (hunk 2 above)
Comment (SaulCM) [reply src/main/java/api/salesforce/Authentication.java:4]: It is required by checkstyle for static imports.

File: src/main/java/core/selenium/WebElementAction.java (hunk 3 above)
Comment (SaulCM) [reply src/main/java/core/selenium/WebElementAction.java:118]: Done.

**Review by GustavoZHA — APPROVED** [review]

**Review by JorgeCaceres-Jala — APPROVED** [review]

**Pull request merged**